<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfigureTelegram</class>
 <widget class="QDialog" name="DialogConfigureTelegram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Telegram</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>511</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>521</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_sender">
    <attribute name="title">
     <string>Sender</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_sender">
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_test_message">
        <property name="text">
         <string>Test Message</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>To get your organization code please request it at info@wadas.it</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_enable_images">
        <property name="text">
         <string>Enable images in Telegram notification messages</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Organization code:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_org_code">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_receivers">
    <attribute name="title">
     <string>Receivers</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_add_receiver">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Receiver</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_remove_receiver">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>250</y>
       <width>251</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Receiver</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_receivers"/>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_errorMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_enable_telegram_notifications">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable Telegram notifications</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NOTE: Telegram name and logo are copyright of Telegram LLC.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogConfigureTelegram</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogConfigureTelegram</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
